fitbit-react/src/Pages/Authentication/Login/Loginform.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react";
+import {
+  AiOutlineMail,
+  AiFillEyeInvisible,
+  AiFillEye,
+} from "../../../Assets/icons";
+import { PrimaryButton, SecondaryButton } from "../../Cart/HorizontalCard";
+import { Link } from "react-router-dom";
+function Loginform() {
+  const [formDetails, setFormDetails] = useState({
+    email: "",
+    password: "",
+  });
+  const [isPasswordVisible, setPasswordVisible] = useState(false);
+  return (
+    <div className="authentication-form-container">
+      <form className="authentication-form">
+        <h2 className="font-large weight-semi-bold authentication-form-heading">
+          <span className="logo-text font-large">FiT</span>BiT
+        </h2>
+        <p className="font-medium-large weight-semi-bold authentication-form-sub-heading">
+          LOGIN INTO YOUR ACCOUNT
+        </p>
+        <div className="form-input-box">
+          <input
+            type="email"
+            placeholder="Email Address"
+            value={formDetails.email}
+            onChange={(e) =>
+              setFormDetails({ ...formDetails, email: e.target.value })
+            }
+          />
+          <AiOutlineMail className="mail-icon" />
+        </div>
+        <div className="form-input-box">
+          <input
+            type={isPasswordVisible ? "text" : "password"}
+            placeholder="Password"
+            value={formDetails.password}
+            onChange={(e) =>
+              setFormDetails({ ...formDetails, password: e.target.value })
+            }
+          />
+          {isPasswordVisible ? (
+            <AiFillEye
+              className="mail-icon"
+              onClick={() => setPasswordVisible((prev) => !prev)}
+            />
+          ) : (
+            <AiFillEyeInvisible
+              className="mail-icon"
+              onClick={() => setPasswordVisible((prev) => !prev)}
+            />
+          )}
+        </div>
+        <div className="checkbox-container">
+          <span>
+            <input type="checkbox" value="Remember Box" id="remember-box" />
+            <label
+              for="remember-box"
```

Comment:
a suggestion to use htmlFor as i think its recommended for react, if im not wrong its probably throwing a warning in your console.

Reply:
Yes, thanks i missed this part